packages/ui/src/Card.module.scss

```diff
@@ -0,0 +1,36 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  padding: c.$grid * 5;
+  box-shadow: 0 c.$cardShadowSize c.$cardShadowBlur 0 c.$colorShadow;
+  border-radius: c.$borderRadius;
+  background: c.$colorNeutralWhite;
+
+  h3.heading {
+    @include c.typographyH3();
+
+    display: flex;
+    align-items: center;
+    padding: 0;
+    margin: 0 0 c.$grid * 4;
+
+    .icon {
+      line-height: 1;
```

Comment:
nit: I removed the `line-height: 1;` and nothing changed - did you encounter a problem without it?

Reply:
Good catch, this is a leftover from the old Card design 🙏